metadata.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name":         "theforeman-concat_native",
+  "version":      "1.3.2",
+  "author":       "theforeman",
+  "summary":      "Native type for multi-part file concatenation",
+  "license":      "Apache-2.0",
+  "source":       "git://github.com/theforeman/puppet-concat",
+  "project_page": "https://github.com/theforeman/puppet-concat",
```

Comment:
Should we rename the repo to "puppet-concat_native"?  It just confused a few folks on #puppet-dev, so now would be a good time if we did.